src/folder/views/Folder/compositions/tree.ts

```diff
@@ -48,14 +48,16 @@ export default function useTree(
   }
 
   function dropExpandedChildren(id: string) {
-    expandedItems = expandedItems.filter(it => it !== id);
+    if (expandedItems.find(it => it === id) != null) {
```

Comment:
We always use type comparison, so for consistency, it would be better to use `!==` instead of `!=`
Or even better, just use `if (expandedItems.find(it => it === id))`, if something is found, this conditional will be evaluated to true.